README.md

```diff
@@ -63,3 +63,50 @@ In the image below you can find some score examples.
 More about ten pin bowling here: http://en.wikipedia.org/wiki/Ten-pin_bowling
 
 ![Ten Pin Score Example](images/example_ten_pin_scoring.png)
+
+### Planning
+
+#### I/O Notes
+
+Input: An array of ten frames, each of which is an array of two rolls
+Output: The sum total of points during that game
+
+Input: A score of ten, i.e. a ‘strike’ has occured during a roll
+Output: The points from the next two rolls are added to the total score
+
+Input: A score of ten has occured during a frame, i.e. a spare
+Output: The points from the next roll is added to the total score
+
```

Comment:
These are good and useful. Would be cool to also have a design image if you had one